opal/browser/dom/node.rb

```diff
@@ -235,17 +230,19 @@ def cdata?
   # @!attribute [r] child
   # @return [Node?] the first child of the node
   def child
```

Comment:
Shouldn't this and all the children related methods in `ParentNode` now?

Also I'm not entirely sure I like the separation of that into a module, what's the rationale behind it?